src/utils/components/SSHSecretSection/components/SecretDropdown/SecretDropdown.tsx

```diff
@@ -1,40 +1,40 @@
 import React, { FC, ReactElement, useState } from 'react';
 
+import { useInstanceTypeVMStore } from '@catalog/CreateFromInstanceTypes/state/useInstanceTypeVMStore';
+import { instanceTypeActionType } from '@catalog/CreateFromInstanceTypes/state/utils/types';
 import { IoK8sApiCoreV1Secret } from '@kubevirt-ui/kubevirt-api/kubernetes';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { getName } from '@kubevirt-utils/resources/shared';
 import { validateSSHPublicKey } from '@kubevirt-utils/utils/utils';
 import { WatchK8sResult } from '@openshift-console/dynamic-plugin-sdk';
 import { Alert, AlertVariant, Select, SelectOption, SelectVariant } from '@patternfly/react-core';
 
-import Loading from '../../../../Loading/Loading';
-import { decodeSecret } from '../../utils';
+import Loading from '../../../Loading/Loading';
+import { decodeSecret } from '../../utils/utils';
 
 type SecretDropdownProps = {
   secretsResourceData: WatchK8sResult<IoK8sApiCoreV1Secret[]>;
-  selectedSecretName: string;
-  onSelectSecret: (secretName: string) => void;
   id?: string;
 };
 
-const SecretDropdown: FC<SecretDropdownProps> = ({
-  secretsResourceData,
-  selectedSecretName,
-  onSelectSecret,
-  id,
-}) => {
+const SecretDropdown: FC<SecretDropdownProps> = ({ secretsResourceData, id }) => {
   const { t } = useKubevirtTranslation();
   const [isOpen, setIsOpen] = useState(false);
 
-  const [allSecrets, secretsLoaded, secretsError] = secretsResourceData;
+  const { instanceTypeVMState, setInstanceTypeVMState } = useInstanceTypeVMStore();
+
```

Comment:
tiny nitpick: These lines are related, so I think you can remove the empty line here.